nemo_aligner/algorithms/ppo.py

```diff
@@ -510,17 +493,18 @@ def fit(self):
                 critic_train_loop_amount = self.critic_warmup_steps + 1 if self.step == 0 else 1
 
                 for _ in range(critic_train_loop_amount):
-                    self.timer.start("rollout_time")
-                    clear_memory()
-                    ppo_rollout_data, metrics, timer_metrics = self.generate_rollouts()
-                    timing_metrics["rollout_time"] = self.timer.stop_and_get_time("rollout_time")
+                    with self.timer("rollout_time"):
+                        clear_memory()
```

Comment:
nit: maybe move the clear memory out of the rollout time

Reply:
done in 73f737180aeb5d4afc72140fc66f33d1544693b0